Sources/Plot/API/HTMLFormEnctype.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  HTMLFormEnctype.swift
+//  
+//
+//  Created by Michael Critz on 12/19/19.
+//
+
+import Foundation
+
+/// Describes the Media Type (also known as MIME type) of a form that defines the serialization of the form.
+/// The actual encoding should be done by the browser / client. This enum merely describes the desired form behavior as HTML.
+/// See HTML spec 4.10.21.3
+/// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#text/plain-encoding-algorithm
+public enum HTMLFormEnctype: CustomStringConvertible {
+    case applicationURLEncoded, multipartFormData, textPlain
+    
+    /// One of `applicationURLEncoded` (the default), `multipartFormData` (for use with files), or `textPlain` (conventionally used for debugging)
+    public var description: String {
```

Comment:
Why not just use 

```swift
public enum HTMLFormEnctype: String, RawRepresentable {
    case applicationURLEncoded = "application/x-www-form-urlencoded"
    case multipartFormData = "multipart/form-data"
    case textPlain = "text/plain"
}
```

which results in a lot less code.
Also I wouldn't use the `description` property for reading the enum's value. https://developer.apple.com/documentation/swift/string/2431307-description

Reply:
Updated to address feedback